EDScoutWebUI/static/EdScout.js

```diff
@@ -349,6 +364,14 @@
         row.id = systemInfo.SystemAddress;
         let chartedSpecificStyle = systemInfo.charted?"charted":"uncharted";
 
+        distance = getDistanceFromCurrentSystem(systemInfo);
+        if (distance > 0) {
+            let systemDistance = document.createElement('div');
+            systemDistance.classList.add("col-1", "system", chartedSpecificStyle, "text-right");
+            systemDistance.innerText = distance.toFixed(2);

```

Comment:
Would be less cluttered to  leave this in whole numbers.

Reply:
Updated in following commit.